boxoffice/views/order.py

```diff
@@ -197,6 +198,7 @@ def payment(order):
                 line_item.discount_coupon.update_used_count()
                 db.session.add(line_item.discount_coupon)
         db.session.commit()
+        boxofficeq.enqueue(stats.post_stats, order.item_collection_id, '#customersupport')
```

Comment:
The room can't be hard-coded. The `Organization` model has a `details` JSON field, which can for instance, hold the name of the slack channel. So, you can say `Organization.details.get('slack_channel')` instead of `#customersupport`.

Also, you can check if this value exists and _then_ add it to the queue. It's possible that an `organization` does not have a slack channel.